decomp/semantics/uds.py

```diff
@@ -1158,11 +1127,12 @@ def __getitem__(self, k):
         return super().__getitem__(k)
 
 class RawUDSDataset(UDSAnnotation):
-    """A raw Universal Decompositional Semantics dataset. Unlike
-       NormalizedUDSDataset, RawUDSDataset may have multiple annotations for
-       a particular attribute. Each annotation is associated with
-       an annotator ID, and different annotators may have annotated
-       different numbers of items.
+    """A raw Universal Decompositional Semantics dataset
+
+       Unlike NormalizedUDSDataset, RawUDSDataset may have multiple
```

Comment:
Expanded descriptions align with the quotes (see [docstring-standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard))